### ISSUE TITLE
[Bug Report] Navigation-drawer background color change

### ISSUE THREAD

**Issue by Noreu** [vuetifyjs/vuetify#11173]
### Environment
**Vuetify Version:** 2.2.22
**Vue Version:** 2.6.11
**Browsers:** Firefox 75.0
**OS:** Windows 10

### Steps to reproduce
Unfortunatly I can't reproduce the bug on codepen.
It's work on dev build with npm run serve, but doesn't work on production build with npm run build.

### Expected Behavior
Navigation drawer with a grey background

### Actual Behavior
Navigation drawer with a black background

### Reproduction Link
<a href="https://codepen.io/Noreu/pen/RwWGmbR" target="_blank">https://codepen.io/Noreu/pen/RwWGmbR</a>

### Other comments
In production the .theme--dark.v-sheet background-color style override the .v-navigation-drawer .v-list background style and apply the the #1e1e1e color in place of inherit attribute.

I have some [mini-css-extract-plugins] error when I run build, for example :

> chunk chunk-88ba3232 [mini-css-extract-plugin]
> Conflicting order. Following module has been added:
>  * css ./node_modules/css-loader/dist/cjs.js??ref--9-oneOf-3-1!./node_modules/postcss-loader/src??ref--9-oneOf-3-2!./node_modules/sass-loader/dist/cjs.js??ref--9-oneOf-3-3!./node_modules/vuetify/src/components/VSheet/VSheet.sass
> despite it was not able to fulfill desired ordering with these modules:
>  * css ./node_modules/css-loader/dist/cjs.js??ref--9-oneOf-3-1!./node_modules/postcss-loader/src??ref--9-oneOf-3-2!./node_modules/sass-loader/dist/cjs.js??ref--9-oneOf-3-3!./node_modules/vuetify/src/directives/ripple/VRipple.sass
>    - couldn't fulfill desired order of chunk group(s) , :
>    - while fulfilling desired order of chunk group(s) , , , , , ,
>  * css ./node_modules/css-loader/dist/cjs.js??ref--9-oneOf-3-1!./node_modules/postcss-loader/src??ref--9-oneOf-3-2!./node_modules/sass-loader/dist/cjs.js??ref--9-oneOf-3-3!./node_modules/vuetify/src/components/VProgressLinear/VProgressLinear.sass
>    - couldn't fulfill desired order of chunk group(s) ,
>    - while fulfilling desired order of chunk group(s) , , , , , ,
>  * css ./node_modules/css-loader/dist/cjs.js??ref--9-oneOf-3-1!./node_modules/postcss-loader/src??ref--9-oneOf-3-2!./node_modules/sass-loader/dist/cjs.js??ref--9-oneOf-3-3!./node_modules/vuetify/src/components/VCard/VCard.sass
>    - couldn't fulfill desired order of chunk group(s) ,
>    - while fulfilling desired order of chunk group(s) , , , , , ,

I'm not comfortable with webpack but I view a tip wich tell to add the code above in vue.config.js :

```
css: {
    extract: {
      ignoreOrder: true
    },
  },
```

Warnings disapears but that don't resolve the problem.

Thanks for the help !

**Comment by jacekkarczmarczyk** [comment]
We accept minimal github repo as reproduction as well

**Comment by Noreu** [comment]
Tried to reproduce the error but I can't.
Don't know why it's work with a smaller project but the warnings are still there during the build...

**Comment by jacekkarczmarczyk** [comment]
You can send screenshots of markup + css both for development and production, similar like in https://github.com/vuetifyjs/vuetify/issues/8067#issuecomment-515696566

**Comment by Noreu** [comment]
The good way :
![navigation-drawer_right](https://user-images.githubusercontent.com/13471804/79862941-2c243880-83d7-11ea-88f9-5ddc5e467c54.JPG)

The wrong :
![navigation-drawer_false](https://user-images.githubusercontent.com/13471804/79862957-37776400-83d7-11ea-9e67-246f20a3a48e.JPG)

I just see it's another css file that override the .theme--dark.v-sheet css rules.

**Comment by yesworld** [comment]
I faced the same problem.. (

**Comment by Noreu** [comment]
I solved the problem with this line on the vue.config.js file :

`lintOnSave: false,
  runtimeCompiler: true,
  filenameHashing:false,
  productionSourceMap: false,
  configureWebpack: {
    resolve: {
       symlinks: false
    },
    optimization: {
      splitChunks: {
       cacheGroups: {
          app: {
            chunks: 'all',
            name: 'main',
            test: /[\\/]src[\\/](.*)[\\/]/,
          },
          flagIcons: {
            chunks: 'all',
            name: 'flagIcons',                
           //test: /[\\/]node_modules[\\/]@coreui[\\/]((icons).*)[\\/]/,
            test: /[\\/]node_modules[\\/]@coreui[\\/]icons[\\/]js[\\/]((flag).*)[\\/]/,
          },
          freeIcons: {
            chunks: 'all',
            name: 'freeIcons',                
           //test: /[\\/]node_modules[\\/]@coreui[\\/]((icons).*)[\\/]/,
            test: /[\\/]node_modules[\\/]@coreui[\\/]icons[\\/]js[\\/]((free).*)[\\/]/,
          },
          brandIcons: {
            chunks: 'all',
            name: 'brandIcons',                
            test: /[\\/]node_modules[\\/]@coreui[\\/]icons[\\/]js[\\/]((brand).*)[\\/]/,
          },
          vendors: {
            chunks: 'all',
            name: 'vendors',
            test: /[\\/]node_modules[\\/]@coreui[\\/]((?!icons).*)[\\/]/,
          },
          // Merge all the CSS into one file
          styles: {
            name: 'styles',
            test: /\.s?css$/,
            chunks: 'all',
            minChunks: 1,
            reuseExistingChunk: true,
            enforce: true,
          }
        }
      }
    }
  }`

**Comment by matheuschimelli** [comment]
> I solved the problem with this line on the vue.config.js file :
> 
> ```
> lintOnSave: false,
>   runtimeCompiler: true,
>   filenameHashing:false,
>   productionSourceMap: false,
>   configureWebpack: {
>     resolve: {
>        symlinks: false
>     },
>     optimization: {
>       splitChunks: {
>        cacheGroups: {
>           app: {
>             chunks: 'all',
>             name: 'main',
>             test: /[\\/]src[\\/](.*)[\\/]/,
>           },
>           flagIcons: {
>             chunks: 'all',
>             name: 'flagIcons',                
>            //test: /[\\/]node_modules[\\/]@coreui[\\/]((icons).*)[\\/]/,
>             test: /[\\/]node_modules[\\/]@coreui[\\/]icons[\\/]js[\\/]((flag).*)[\\/]/,
>           },
>           freeIcons: {
>             chunks: 'all',
>             name: 'freeIcons',                
>            //test: /[\\/]node_modules[\\/]@coreui[\\/]((icons).*)[\\/]/,
>             test: /[\\/]node_modules[\\/]@coreui[\\/]icons[\\/]js[\\/]((free).*)[\\/]/,
>           },
>           brandIcons: {
>             chunks: 'all',
>             name: 'brandIcons',                
>             test: /[\\/]node_modules[\\/]@coreui[\\/]icons[\\/]js[\\/]((brand).*)[\\/]/,
>           },
>           vendors: {
>             chunks: 'all',
>             name: 'vendors',
>             test: /[\\/]node_modules[\\/]@coreui[\\/]((?!icons).*)[\\/]/,
>           },
>           // Merge all the CSS into one file
>           styles: {
>             name: 'styles',
>             test: /\.s?css$/,
>             chunks: 'all',
>             minChunks: 1,
>             reuseExistingChunk: true,
>             enforce: true,
>           }
>         }
>       }
>     }
>   }
> ```

What line specifically?

**Comment by matheuschimelli** [comment]
I'm having the same problem with v-drawer. I tried to update vuetify to the last stable version but it didn't work. I'm suspecting the problem itself is with the color property on v-drawer that is not been applied because the `theme--dark.v-sheet` class is overwriting the `color` property

![image](https://user-images.githubusercontent.com/10470074/85213138-bc77ee80-b330-11ea-93fe-76834c43de0e.png)

The `color` property is applied, but it has not effect
![image](https://user-images.githubusercontent.com/10470074/85213170-1ed0ef00-b331-11ea-8217-52dbf9ee3aa3.png)

I could solve it by adding a class on `v-list`:

```html
<v-list class="customListColor">
```

```css
.customListColor {
  background: rgb(55, 71, 79) !important;
}
```
It solved my problem.

**Comment by KaelWD** [comment]
`<v-list color="transparent">`

**Comment by johnleider** [comment]
fixed in baeb8a692
